src/util.js

```diff
@@ -401,8 +401,22 @@
       }
 
       return md5(str);
-    }
+    },
     /* jshint ignore:end */
+
+
+    localStorageAvailable: function() {
+      if (!('localStorage' in global)) { return false }
```

Comment:
This introduces an error when I'm building RS.js from this PR and use it in my favorite drinks:

```
ReferenceError: Can't find variable: global
```